carmajava/launch/cadillac_config/Dockerfile

```diff
@@ -0,0 +1,20 @@
+#  Copyright (C) 2018 LEIDOS.
+# 
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy of
+#  the License at
+# 
+#  http://www.apache.org/licenses/LICENSE-2.0
+# 
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations under
+#  the License.
+
+FROM busybox:latest
+
+ADD . /root/vehicle
+VOLUME /opt/carma/vehicle
+
+CMD  cp /root/vehicle/* /opt/carma/vehicle
```

Comment:
Is this still from /root or should it now be /home/carma or ~ ?

Reply:
Since this is a different image this is still correct. This image is based on the `busybox` image instead of `carma-base` as the `busybox` image is much smaller (cadillac-config is 1.17 MB total, data and all, compared to just `carma-base`'s 1.59 GB) only containing a shell and the barest essential POSIX tools.

Since this image serves only the purpose of starting up and copying some files, it's sufficient to just use the root user (note: not actually the hosts's root, but still has full R/W to the containerized filesystem)